requirements.txt

```diff
@@ -5,7 +5,7 @@
 # requirements-build.txt
 
 # In-house dependencies
-leaf-common==1.2.22
+leaf-common>=1.2.23
```

Comment:
Use latest leaf-common.
Also use >= so this does not need to be an impediment in the future. leaf-common is stable enough these days.